fudge-ratio/src/main/webapp/script.js

```diff
@@ -66,22 +68,62 @@ buttonClear.addEventListener("click", () => {
   if (button.textContent === "Stop") button.click();
   (seconds = 0), (minutes = 0), (hours = 0);
   updateText();
+  cleanSessionState();
 });
 
 const formatTaskString = () => {
-    let dropdown = document.getElementById("dropdown-contents");
-    let op = dropdown.options[dropdown.selectedIndex];
-	let category = op.parentNode.label;
-    let formattedString = "";
-    if (category == "Schoolwork"){
-        formattedString = task + " homework!";
-    }
-    else if (category == "Job-related tasks"){
-        formattedString = "doing " + task + "!" ;
-    }
-    else formattedString = task + "!"; 
-    return formattedString; 
-}
+  let op = dropdown.options[dropdown.selectedIndex];
+  let category = op.parentNode.label;
+  let formattedString = "";
+  if (category == "Schoolwork") {
+    formattedString = task + " homework!";
+  } else if (category == "Job-related tasks") {
```

Comment:
I'm not sure about the other teammates, but I put the closing braces on a separate line because I find it a lot easer to read (it's also just a habit for me). I guess it would be best for us to all stick to one style, so maybe we can see what the majority is and then go by that?

Reply:
I totally agree with you, we should set guidelines for things like this.
In Javascript there is a tool called Prettier (one of many) and his job is to take some code and apply just stylistic modifications.

https://prettier.io/

So the idea is that we have a configuration file were we put how we want our code to be "formated" (spaces vs tabs, semicolons)and the tool do the rest of the work.

Here is all the possible configurations we could use:
https://prettier.io/docs/en/options.html

Reply:
What do you think @fancast , @gilemos?